website/data/intro-nav-data.json

```diff
@@ -29,27 +29,23 @@
     "routes": [
       {
         "title": "Overview",
-        "href": "https://learn.hashicorp.com/collections/nomad/get-started"
+        "href": "/nomad/tutorials/get-started"
       },
       {
         "title": "Running Nomad",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-run"
+        "href": "/nomad/tutorials/get-started/get-started-run"
       },
       {
         "title": "Jobs",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-jobs"
-      },
-      {
-        "title": "Clustering",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-cluster"
+        "href": "/nomad/tutorials/get-started/get-started-jobs"
       },
```

Comment:
Ended up removing this one because the link 404s and this tutorial also isn't present in [the same collection](https://learn.hashicorp.com/collections/nomad/get-started) as all of the other tutorials listed here.